modules/ROOT/pages/login-platform-from-studio.adoc

```diff
@@ -0,0 +1,18 @@
+= Login to Anypoint Platform from Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Enhance your security in Anypoint Studio by configuring two-factor authentication through xref:access-management::multi-factor-authentication.adoc[Anypoint Platform]. 
```

Comment:
```suggestion
Enhance your security in Anypoint Studio by xref:access-management::multi-factor-authentication.adoc[configuring two-factor authentication] through Anypoint Platform. 
```
**Optional:** I just suggest this change because the target article seems to explain how to configure 2FA and not what Anypoint Platform is, right?

Reply:
OK! I like it! Thanks!